resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/KubernetesSuite.scala

```diff
@@ -275,7 +275,7 @@ private[spark] class KubernetesSuite extends SparkFunSuite with BeforeAndAfter {
     assert(jobLog.contains("Pi is roughly"), "Pi was not computed by the job...")
   }
 
-  test("Run with custom labels") {
+  test("Run with custom labels and custom ports") {
```

Comment:
On the other hand the test space is growing pretty large. Eventually `KubernetesSuite` should be broken up anyways.